src/ray/util/tests/pipe_logger_test.cc

```diff
@@ -41,28 +49,27 @@ TEST_P(PipeLoggerTest, NoPipeWrite) {
          absl::StrFormat("%d", pipe_buffer_size).data(),
          /*overwrite=*/1);
 
```

Comment:
This test is the same as `RedirectionWithNoTeeAndRotation`?

Reply:
Yes, but that one's for unix only.

Reply:
I'm confused. What's the difference between this test and the other test? Both don't have pipe.

Reply:
This particular test case serves as a black-box test:
- Pipe might or might not be involved;
- Pipe size might affect performance and behavior;

So different from the cross-platform one, apart from the general logging content check, it also tests on pipe size and env setting.

I didn't expect strong opposition TBH, if you don't think it necessary, it's been removed in the latest commit.

Reply:
Actually what I mean is that you can just run NoPipeWrite on windows: as you said, pipe is not interface, so as long as stream redirection opt is supported by windows, NoPipeWrite should also pass on windows?

Reply:
ok